src/ast/passes/codegen_llvm.cpp

```diff
@@ -3513,18 +3513,25 @@ void CodegenLLVM::createMapDefinition(const std::string &name,
   var->setDSOLocal(true);
   var->addDebugInfo(debuginfo);
 }
+bool CodegenLLVM::needs_percpu_map(const SizedType &val_type)
+{
+  return ((val_type.IsHistTy() || val_type.IsLhistTy() ||
+           val_type.IsCountTy() || val_type.IsSumTy() || val_type.IsMinTy() ||
+           val_type.IsMaxTy() || val_type.IsAvgTy() || val_type.IsStatsTy()))
+             ? true
+             : false;
```

Comment:
nit: You don't need this ternary and these parens.
```suggestion
  return val_type.IsHistTy() || val_type.IsLhistTy() ||
           val_type.IsCountTy() || val_type.IsSumTy() || val_type.IsMinTy() ||
           val_type.IsMaxTy() || val_type.IsAvgTy() || val_type.IsStatsTy();
```